### PR TITLE
Installs Titiler as a submodule.

### PR DESCRIPTION
**Pull request by paulpilone**

### CODE REVIEW

**Comment by paulpilone** [comment]
Added Titiler as a submodule which installs from the E84 hosted Helm repository. I also updated the READMEs with instructions on how to install and verify installation. Since Titiler doesn't currently support ARM architecture I added a flag in the root inputs.tf you can use to disable the installation of Titiler.

**Review by hectormachin — APPROVED** [review]
This worked for me, great job on the module structure!!
Screenshot 2023-04-12 at 10 05 37 AM
Screenshot 2023-04-12 at 10 05 58 AM

File: README.md [hunk 1]
```diff
@@ -245,6 +245,14 @@ For accessing application 2, you can go to: `http://hello-world.local:{NGINX_NOD
 
 The local development environment includes resources for aggregating and visualizing logs and metrics. More information is available [here](./modules/monitoring/README.md).
 
+### Tiling
+
+The local development environment incldues resources for rendering and serving tiles. More information is available [here](./modules/tiling/README.md).
```
Comment (geomatician) [inline README.md:250]: 'includes'

Comment (paulpilone) [reply README.md]: @geomatician fixed!

**Pull request merged**